apis_core/collections/signals.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from crum import get_current_request
+from apis_core.collections.models import SkosCollection, SkosCollectionContentObject
+from apis_core.history.models import VersionMixin
+from django.contrib import messages
+
+
+@receiver(post_save)
+def add_to_session_collection(
+    sender, instance, created, raw, using, update_fields, **kwargs
+):
+    """
+    Add a created object to all the SkosCollections that are listed
+    in the `session_collections` session variable.
+    """
+    request = get_current_request()
+    if isinstance(instance, VersionMixin) and request:
```

Comment:
This check here never hits. The instance is of `VersionPerson` rather than `VersionMixin`

Reply:
so when changing something in a Person there is two calls to this signal, one with `isinstance(instance, Person)` and one with `isinstance(instance, VersionPerson)`

Reply:
```suggestion
    if hasattr(instance, "history_id") and request:
```
would allow to further down just use the object as is (the history object just created) instead of hitting the DB for retrieving the latest history object.